File: app/src/main/java/com/example/randomizer/MainActivity.java
package com.example.randomizer;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.kostasdrakonakis.randomizer.Randomizer;
import com.kostasdrakonakis.randomizer.annotations.RandomInt;
import com.kostasdrakonakis.randomizer.annotations.RandomString;

public class MainActivity extends AppCompatActivity {

    @RandomInt
    int number;

    @RandomString
    String rand;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Randomizer.bind(this);
        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/example/randomizer/MainActivity.kt
package com.example.randomizer

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.kostasdrakonakis.randomizer.Randomizer
import com.kostasdrakonakis.randomizer.annotations.RandomInt
import com.kostasdrakonakis.randomizer.annotations.RandomString

class MainActivity : AppCompatActivity() {
    @RandomInt
    @JvmField
    var number: Int = 0

    @RandomString
    lateinit var rand: String

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Randomizer.bind(this)
        setContentView(R.layout.activity_main)
    }
}